Frontend/CO.CDP.OrganisationApp/Pages/Organisation/OrganisationEmail.cshtml

```diff
@@ -49,7 +49,7 @@
 
                 <button type="submit"
                         class="govuk-button"
-                        data-module="govuk-button">
+                        data-prevent-double-click="true" data-prevent-double-click="true" data-module="govuk-button">
```

Comment:
`data-prevent-double-click="true"` is doubled here.

Reply:
well spotted - all updated now